test/foundry/unit/StakingRewardsV2/StakingRewardsV2.t.sol

```diff
@@ -1142,54 +1053,25 @@ contract StakingRewardsV2Test is DefaultStakingV2Setup {
         // check balance decreased
         assertEq(initialEscrowBalance - 1 weeks, finalEscrowBalance);
     }
-
-    function test_unstakeEscrowSkipCooldown_Does_Change_Escrowed_Balances_Mapping() public {
```

Comment:
**Though:** Can we have an equivalent of this test with `unstakeEscrowAdmin`?

Reply:
Added `test_unstakeEscrowAdmin_Does_Change_Escrowed_Balances_Mapping`